docs/devices/WXKG16LM.md

```diff
@@ -60,6 +60,8 @@ To read (`/get`) the value publish a message to topic `zigbee2mqtt/FRIENDLY_NAME
 To write (`/set`) a value publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/set` with payload `{"click_mode": NEW_VALUE}`.
 The possible values are: `fast`, `multi`.
 
+Note: You may need to press button 5 times in order to change click mode.
```

Comment:
Thanks but this part of the docs is generated, could you update: https://github.com/Koenkk/zigbee-herdsman-converters/blob/5009796f1c240de3ac2d59ff9302cb0577a5280b/src/devices/lumi.ts#L3670 instead?